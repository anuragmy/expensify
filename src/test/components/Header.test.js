import ReactShallowRenderer from "react-test-renderer/shallow";
import Header from "../../components/Header";
import React from "react";

test("should render header ccorrectly", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  console.log(renderer.getRenderOutput());
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
