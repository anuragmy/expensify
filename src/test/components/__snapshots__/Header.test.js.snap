// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header ccorrectly 1`] = `
<header>
  <h1
    className="text-center text-primary "
  >
    Expensify
  </h1>
  <div
    className="navigation"
  >
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/create"
    >
      Create Expense
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/help"
    >
      Help
    </NavLink>
  </div>
</header>
`;
